src/scripts/helpers/zotonic_setup

```diff
@@ -40,8 +40,7 @@ export -f require_zotonic_not_running
 function enablesite {
     config="priv/sites/$1/config"
     expr="s/enabled, \(true\|false\)/enabled, $2/"
-    perl -e "$expr" -pi $config
-    #echo sed -e "$expr" -i $config
+    sed -e "$expr" -i.bak -e $config
```

Comment:
sed -i with an extension works on the platforms I've tried (sunos, osx, linux) but it don't see the 'enablesite' function actually being called anywhere, plus it hardcodes the sites dir as being under priv/ so I didn't give it as much thought as the main config file. I actually considered removing it entirely.